araproc/analysis/standard_reco.py

```diff
@@ -173,14 +211,20 @@ def do_standard_reco(self, waveform_bundle):
 
         Parameters
         ----------
-        waveform_bundle : dict
-            A dictionary of the 16 waveforms.
-            The key is the RF channel number.
-            The value is a TGraph.
-            There should be 16 entries, even if you don't intend to use all
-            16 traces in your interferometry.
-            The exclusions are handled further down under the excluded channels section.
-        
+        wavepacket : dict
+            A dict with three entries:
+              "event" : int  
+                Event number
+              "waveforms" : dict
+                A dictionary of the 16 waveforms.
+                The key is the RF channel number.
+                The value is a TGraph.
+                There should be 16 entries, even if you don't intend to use all
+                16 traces in your interferometry.
+                The exclusions are handled further down under the excluded channels section.
+              "type" : string
```

Comment:
This is nitpicky, but `type` is a protected work in python. (like `type` means something, e.g. `type(some_variable)`. Can we pick another word?

Reply:
Fixed!